src/Tribe/Views/V2/Widgets/Widget_List.php

```diff
@@ -72,6 +73,56 @@ class Widget_List extends Widget_Abstract {
 		],
 	];
 
+	/**
```

Comment:
* :no_entry_sign: Doc comment for parameter "$context" missing
* :no_entry_sign: Doc comment for parameter "$view" missing

<sub>hash: c4ef70200ceaf627978404b6cdff6eb9</sub>